std/mem.zig

```diff
@@ -355,6 +382,10 @@ pub fn startsWith(comptime T: type, haystack: []const T, needle: []const T) bool
     return if (needle.len > haystack.len) false else eql(T, haystack[0 .. needle.len], needle);
 }
 
+pub fn endsWith(comptime T: type, haystack: []const T, needle: []const T) bool {
```

Comment:
can you add a test for this?

Reply:
Sure :).  I'll add a test for starts with too.